test/ml/test_ml.py

```diff
@@ -48,3 +48,14 @@ def test_hac_vertical(self):
 
         assert filecmp.cmp(OUT_DIR + 'hac-clusters-vertical.txt', EXPECT_DIR + 'expected-hac-vertical-clusters.txt')
 
+    def test_ensemble_network(self):
+        dataframe = ml.summarize_networks([INPUT_DIR + 'test-data-s1/s1.txt', INPUT_DIR + 'test-data-s2/s2.txt', INPUT_DIR + 'test-data-s3/s3.txt'])
+        ml.ensemble_network(dataframe, OUT_DIR + 'ensemble-network.csv')
```

Comment:
I prefer only using the `.csv` file extension for comma-separated files with actual commas as the separators. Can we change back to `.txt` or `.tsv` for tab-separated, which is also fairly common? We'll need to change the expected file below as well.